.circleci/config.yml

```diff
@@ -14,8 +14,43 @@ jobs:
           wd: ~/repo
       - flake8/flake8_errors:
           wd: ~/repo
+  server-tests:
+    docker:
+      - image: circleci/python:3.8
+        environment:
+          PIPENV_VENV_IN_PROJECT: true
+      - image: docker.elastic.co/elasticsearch/elasticsearch:7.6.2
+        environment:
+          - cluster.name: elasticsearch
+          - xpack.security.enabled: false
+          - transport.host: localhost
+          - network.host: 127.0.0.1
+          - discovery.type: single-node
+    working_directory: ~/repo/natlas-server
+    steps:
+      - checkout:
+          path: ~/repo
+      - restore_cache:
+          name: Restore pipenv cache
+          keys:
+            - pipenv-v0-{{ checksum "Pipfile.lock" }}
+      - run:
+          name: Install Pipenv
+          command: |
+            pip install pipenv
+            pipenv install --dev
```

Comment:
One thing we might care about is whether or not we want to pipenv install the locked version (a la `pipenv sync`) . I'm leaning towards leaving it like this, because if a package updates and causes backwards incompatibility, we'll end up with a failed test, which lets us catch the problem sooner while maintaining support for latest versions, generally speaking.

Reply:
I was not aware of this difference. CI's should always install packages that are specified. It'd be extremely confusing if our CI is running code different than what's checked in to our code base.

Instead, we'll use dependabot to safely guarantee new dependency updates will succeed.